### ISSUE TITLE
.docx: the cell of this table has no hatched background

### ISSUE THREAD

**Issue by parrenin** [ONLYOFFICE/DocumentServer#1503]
**Do you want to request a *feature* or report a *bug*?**
Report a bug.

**What is the current behavior?**
In the attached .docx file, there is a table containing a cell with a hatched background.
This background does not show up in OO.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
1. Open the attached .docx file in Word
2. Open the same file in OO
3. Compare

**What is the expected behavior?**
The table should appear the same way in Word and OO.

**Did this work in previous versions of DocumentServer?**
No.

**DocumentServer version:**
6.4.1.

**Operating System:**
Linux

**Browser version:**
Firefox 94.

**Comment by parrenin** [comment]
This is the .docx file showing the problem:

[bug-OO-hachure.docx](https://github.com/ONLYOFFICE/DocumentServer/files/7497248/bug-OO-hachure.docx)

**Comment by TatianaKirillova** [comment]
Hello, **@parrenin**
Thanks for your request! I can confirm - this is know bug, issue 16400 in our private issue tracker.

**Comment by parrenin** [comment]
Still present in OO 7.0.0

**Comment by ShockwaveNN** [comment]
Yes, we will close issue as soon as our dev team is fixed it, but it's not fixed

**Comment by parrenin** [comment]
Still repro in 7.0.2.

**Comment by parrenin** [comment]
This issue has changed in 8.0.0.
Now, the cell has a grey background, but it is still not hatched.

**Comment by TatianaKirillova** [comment]
@parrenin Yes you are right. As soon as the problem is resolved, we will definitely inform you here in the comments.
